### PR TITLE
#1697748890 setting up morgan logger for https 

### PR DESCRIPTION
**Pull request by b0nbon1**
### What does this PR do?
This PR sets up morgan for logging the HTTP requests that are made to the API
#### Description of Task to be completed?
- [x] install morgan
- [x] setup morgan middleware to log the errors using Winston logger
#### How should this be manually tested?
- git clone the repository
- run `npm install` to install the dependencies
- run `npm run watch`
- go to postman and test using endpoint `localhost:3000` 
- then check on the terminal if it has been logged
#### Any background context you want to provide?
- Winston logger is used to logging
#### What are the relevant pivotal tracker stories?
[#1697748890](https://www.pivotaltracker.com/n/projects/2407417/stories/169774890)
#### Screenshots
image

### CODE REVIEW

File: src/utils/winston.js [hunk 1]
```diff
@@ -1,16 +1,19 @@
 import { createLogger, format, transports } from 'winston';
 
 const {
-  combine, timestamp, simple, json
+  combine, timestamp, simple, json, prettyPrint, colorize
```
Comment (timolinn) [inline src/utils/winston.js]: The `colorize` format seems to be messing up the log files.

**Review by timolinn — CHANGES_REQUESTED** [review]
This seems to be working, but it logs sensitive information to the console. Also, checking the log files, I noticed `winston` isn't working well with the current format (the `id` key and value are logged as the string value for `message`).

**Comment by b0nbon1** [comment]
@timolinn should I remove the request?

**Comment by timolinn** [comment]
> @timolinn should I remove the request?

@b0nbon1 You can check this [implementation](https://github.com/timolinn/node-starter/blob/8f6e79aefde851c548146defac38e274089f80ee/src/app.js#L22) and see how it works including what gets logged. However, I really liked the idea of having an `id` for each request.

File: src/utils/winston.js (hunk 1 above)
Comment (b0nbon1) [reply src/utils/winston.js]: Yeah yea, I have noticed now that the log files. Thanks

**Comment by b0nbon1** [comment]
@timolinn I made the changes and I also added a line to create a `request.log` for keepings the logged requests. Which I thought it is okay  for tracking the logs

**Review by timolinn — DISMISSED** [review]

**Review by gildniy — DISMISSED** [review]
Tested and working.

File: src/app.js [hunk 2]
```diff
@@ -33,6 +39,16 @@ app.use(cors({
...
+// const loggerFormat = '[:date[web]] ":method :url" status: :status :response-time s';
+// app.use(morgan(loggerFormat, {
+//   stream: process.stdout
+// }));
+
+// app.use(async (req, res, next) => {
+//   logger.info(`body: ${JSON.stringify(req.body)}`);
+//   next();
+// });
+
```
Comment (Mastel22) [inline src/app.js]: You can delete this

**Review by Mastel22 — APPROVED** [review]
LGTM

**Review by gildniy — APPROVED** [review]
Working good

**Review by nshaphan — APPROVED** [review]
LGTM

**Review by timolinn — APPROVED** [review]

**Pull request merged**